### PR TITLE
[Feat] Provider down modal UX improvement

### PR DESCRIPTION
**Pull request by antonyayoub**
# Name of the feature/issue
Provider down modal UX improvement
## Description
Provider down modal UX improvement by indicating that it might be a connection problem, adding a restart Wallet button and adding an option to change RPC URL of the selected network if the provider is down. 

We are now using [`chrome.runtime.reload()`](https://developer.chrome.com/docs/extensions/reference/runtime/#method-reload) instead of the `lockApp()` method to reload the whole extension UI and background instead of manually asking the user to do so from the [chrome://extensions](chrome://extensions) page.

https://user-images.githubusercontent.com/47082495/210842016-9dbff893-74cc-4598-9be5-c4729b36261a.mov

### CODE REVIEW

File: packages/ui/src/components/dialog/ProviderDownDialog.tsx [hunk 1]
```diff
@@ -1,33 +1,74 @@
...
+                            optionsContainerClassName="overflow-auto max-h-72"
+                        />
+                        <CgDanger className="text-red-500 w-20 h-20 mb-2 block m-auto" />
+                    </>
+                }
+                title="Provider down"
+                message={
+                    <div className="mb-4">
+                        <p className="mb-2">
+                            The current network is down. Please select another
```
Comment (julianariel) [inline packages/ui/src/components/dialog/ProviderDownDialog.tsx:37]: we should improve this message

**Review by julianariel — COMMENTED** [review]
Looks good to me, nice work! Added a few comments.

Please upload a GIF/Vid of the new modal to the PR description and add a brief explanation about why we implemented `chrome.runtime.reload()` method and/or link to the docs

File: packages/ui/src/router/PopupRouter.tsx [hunk 2]
```diff
@@ -109,7 +110,7 @@ const PopupRouter = ({
     const state = useBlankState()!
     const isOnboarded = state?.isOnboarded
     const resetHandler = async () => {
-        await lockApp()
+        chrome.runtime.reload()
```
Comment (fpezzola) [inline packages/ui/src/router/PopupRouter.tsx:113]: Are we able to access the `chrome.runtime` from the UI? I am not sure about that.

Comment (fpezzola) [reply packages/ui/src/router/PopupRouter.tsx:113]: Seems to work 👍

Comment (antonyayoub) [reply packages/ui/src/router/PopupRouter.tsx:113]: 💪

File: packages/ui/src/components/dialog/ProviderDownDialog.tsx [hunk 3]
```diff
@@ -1,33 +1,74 @@
...
 const ProviderDownDialog = () => {
     const { isProviderNetworkOnline, isUserNetworkOnline } = useBlankState()!
     const location = useLocation()
+    const history = useOnMountHistory()
 
     const showDialog =
         !isProviderNetworkOnline &&
         isUserNetworkOnline &&
-        location.pathname !== "/chain/switch"
+        location.pathname !== "/chain/switch" &&
```
[the remark this answers is not in the document]
Comment (antonyayoub) [reply packages/ui/src/components/dialog/ProviderDownDialog.tsx:17]: Sure that makes more sense!

File: packages/ui/src/components/dialog/ProviderDownDialog.tsx (hunk 1 above)
Comment (antonyayoub) [reply packages/ui/src/components/dialog/ProviderDownDialog.tsx]: 👌

**Review by julianariel — APPROVED** [review]
LGTM 👏🏻

**Pull request merged**